source/index.html.md

```diff
@@ -1075,3 +1075,60 @@ See [Create Report Template](#create-report-template) for details on format and
 ### HTTP Request
 
 `POST/PUT https://portal.cubesoftware.com/api/v1/templates/<id>`
+
+# User Settings
+
+User settings contain various defaults tied to users.
+
+
+## Spreadsheet Settings
+
+```python
+import requests
+
+r = requests.get('<url>', headers={
+    'Authorization': 'example-access-token'
+})
+r.json()
+```
+
+> The above request returns a JSON structured like this:
+
+```json
+{
+    "dimensions": {
+        "filters": [
+            {
+                "top_level_id": 18000,
+                "selected_id": 20270
+            },
+            {
+                "top_level_id": 18003,
+                "selected_id": 18004
+            },
+            {
+                "top_level_id": 18005,
+                "selected_id": 18006
+            },
+            {
+                "top_level_id": 18009,
+                "selected_id": 18027
+            }
+        ]
+    }
+}
+```
+
+This endpoint fetches default values to use in various cases for the user in the context
+of a spreadsheet (i.e. inside of Cube add-ons).
```

Comment:
@joshholat If I'm an outsider or new to the code base and reading this, I wouldn't know what this means or why I'd use it. 

Maybe say something about how this endpoint returns reasonable defaults when the user hasn't specified dimensions to use for rows, columns, filters